docs/src/gs/create-workload-cluster.md

```diff
@@ -8,15 +8,19 @@ Choose one of the available templates for to create your workload clusters from
+| `OCI_CONTROL_PLANE_SHAPE`                 | VM.Standard.E4.Flex | The shape of the Kubernetes nodes                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `OCI_CONTROL_PLANE_SHAPE_MEMORY_IN_GBS`   |                     | The amount of memory to be allocated to the instances. If not provided it is automatically computed by compute API.                                                                                                                                                                                                                                                                                                                                                                   |
+| `OCI_CONTROL_PLANE_SHAPE_OCPUS`           | 1                   | The number of OCPUs allocated to the instance                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `OCI_WORKER_SHAPE`                        | VM.Standard.E4.Flex | The shape of the Kubernetes worker nodes                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `OCI_WORKER_SHAPE_MEMORY_IN_GBS`          |                     | The amount of memory to be allocated to the <br/>instances. If <br/>not provided it is automatically computed by compute API for VMs, but you will need to set this if you want to use Bare Metal instances.                                                                                                                                                                                                                                                                          |
```

Comment:
Please replace with "The amount of memory in GBs to be allocated to the worker instances. If not provided, the memory is automatically computed by compute API."